src/sentry/receivers/sentry_apps.py

```diff
@@ -75,6 +78,30 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             user_id=(user.id if user else None),
             data=data,
         )
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        from sentry.models import SentryFunction
+
+        data["user"] = serialize(User.objects.get(id=user.id), user, UserSerializer())
+        data["issue"] = serialize(Group.objects.get(id=issue.id))
+
+        for fn in SentryFunction.objects.filter(organization=organization).all():
```

Comment:
How many functions can an org have?

Reply:
@markstory Hard to say but I'd be shocked if folks have more than a few per org.